<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalibration</class>
 <widget class="QWidget" name="CameraCalibration">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>551</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>CameraCalibration</string>
  </property>
  <widget class="QPushButton" name="pbSample">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Sample</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Cross in Width of the board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Cross in Height of the board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Number Calibration Views</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbNumCrossW">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>9</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbNumCrossH">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbNumCaliView">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbCalibration">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>240</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbSuccessView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUndistorted">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>240</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Undistorted</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbImgW">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height of the Image for Calibration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbImgH">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width of the Image for Calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCalibration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Get it!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Press SPACE to ACCEPT the View!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Matrix</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,1,3,1,3">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="dsbFx">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="dsbCM12">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="dsbCx">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dsbCM21">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="dsbFy">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="dsbCy">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dsbCM31">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="dsbCM32">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QDoubleSpinBox" name="dsbHomoW">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Distortion Coefficients</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>293</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3,1,3,1,3">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="dsbK1">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="dsbK2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="dsbP1">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dsbP2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="dsbK3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>K1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>K2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>P1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>P2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>K3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSample</sender>
   <signal>clicked()</signal>
   <receiver>CameraCalibration</receiver>
   <slot>pbOnClickSample()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalibration</sender>
   <signal>clicked()</signal>
   <receiver>CameraCalibration</receiver>
   <slot>pbOnClickCalibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbTest</sender>
   <signal>clicked(bool)</signal>
   <receiver>CameraCalibration</receiver>
   <slot>pbOnClickTestCaliResult(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCalibration</receiver>
   <slot>pbOnClickAcceptGoodView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>CameraCalibration</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pbOnClickCalibration()</slot>
  <slot>pbOnClickSample()</slot>
  <slot>pbOnClickTestCaliResult(bool)</slot>
  <slot>pbOnClickAcceptGoodView()</slot>
 </slots>
</ui>
